packages/fxa-auth-server/lib/signer.ts

```diff
@@ -2,15 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+import { JWTool, PrivateJWK } from '@fxa/vendored/jwtool';
 
-const { JWTool } = require('@fxa/vendored/jwtool');
-
-module.exports = function (secretKeyFile, domain) {
-  const key = JWTool.JWK.fromFile(secretKeyFile, { iss: domain });
+module.exports = function (secretKeyFile: string, domain: string) {
```

Comment:
Q: should this be `export default` to match the `import` statement on L5? Or does it matter in .ts files if we mix-and-match syntaxes?

Reply:
I was under the impression that this would avoid having to change anyone importing this, by leaving it the same. But yes, I think in TypeScript, I need to declare this a `export default` to match.